content/applications/finance/accounting/bank/setup/foreign_currency.rst

```diff
@@ -2,116 +2,116 @@
+institutions, according to your configuration.
 
 Configuration
 =============
 
 Activate the multi-currency feature
 -----------------------------------
 
-In order to allow your company to work with multiple currencies, you
-should activate the multi-currency mode. In the accounting application,
-go into :menuselection:`Configuration --> Settings --> Accounting & Finance Features`
-make sure the **Allow Multi-currencies** box is ticked. Provide
-a **Currency Exchange Gain / Loss** account, then click on **Apply**.
+In order to allow your company to work with multiple currencies, you need to activate the
+**multi-currency** mode. In the accounting application, go into :menuselection:`Accounting -->
+Configuration --> Settings --> Currencies` and make sure the :guilabel:`Allow Multi-currencies` box
+is ticked. Provide a **Currency Exchange Gain / Loss** account and :guilabel:`Save`.
 
 Configure currencies
 --------------------
 
-Once the Odoo is configured to support multiple currencies, you should
-activate the currencies you plan to work with. To do that, go to the menu
-:menuselection:`Configuration --> Currencies`. All the currencies are created by default,
-but you should activate the ones you plan to support (to activate a
-currency, check its "Active" field).
-
-After having activated the currencies, you can configure the parameters
-to automate the currency rate update. These options are also in the
-settings of the Accounting application, in the bottom of the page:
+Once Odoo is configured to support multiple currencies, activate the currencies you plan to work
+with. To do that, go into :menuselection:`Accounting --> Configuration --> Currencies` or in
+:guilabel:`Activate Other Currencies` that appears under :guilabel:`Multi-Currencies` once the box
+has been checked. All currencies are created by default, but only a few are active from the start.
 
-.. image:: foreign_currency/foreign01.png
-   :align: center
+Once the currencies are activated, you can choose to **automate** the currency rate update, or leave
+it on **manual**. To configure the rate update, under the same :guilabel:`Currencies` menu, check
+:guilabel:`Automatic Currency Rates` and set :guilabel:`Interval` to your desired frequency. You
+also have the option to choose the :guilabel:`Service` you wish to obtain currency rates from.
 
-Click on the **Update Now** link to update the currency rates now.
+Click on the :guilabel:`Update Now` button to update the currency rates manually.
 
 Create a new bank account
 -------------------------
 
-In the accounting application, we first go to :menuselection:`Configuration -->
-Accounting / Bank account`, and we create a new one.
+In the accounting application, go to :menuselection:`Accounting --> Configuration --> Journals` and
+create a new one. Fill in the different fields, and :guilabel:`Save`.
 
-.. image:: foreign_currency/foreign02.png
+.. image:: foreign_currency/foreign-journal.png
```

Comment:
Resize your browser window so elements appear tighter, then only select a part of the screen, something like this would be more readable already: https://i.imgur.com/nrsrcW5.png